src/main/webapp/resources/js/pages/analyses/analysis-outputs-table.js

```diff
@@ -1,369 +1,371 @@
-//         return `${name} (${version})`;
-//       }
-//     },
-//     {
-//       field: "analysisSubmissionName",
-//       headerName: i18n("analysis-submission"),
-//       cellRenderer: p =>
-//         `<a href="${BASE_URL}analysis/${p.data.analysisSubmissionId}" target="_blank">${p.data.analysisSubmissionName}</a>`
-//     },
-//     PROJECT_ID
-//       ? {
-//           field: "userId",
-//           headerName: i18n("project.export.submitter"),
-//           cellRenderer: p => `${p.data.userFirstName} ${p.data.userLastName}`
-//         }
-//       : null,
-//     {
-//       field: "createdDate",
-//       headerName: i18n("analysis.date-created"),
-//       cellRenderer: p => formatDate({ date: p.data.createdDate })
-//     }
-//   ].filter(header => header !== null);
-//   const $grid = newElement(
-//     `<div id="grid-outputs" class="ag-theme-balham" style="height: 600px; width: 100%; resize: both;"/>`
-//   );
-//   const $dlButton = newElement(
-//     `<button type="button" class="btn spaced-bottom" disabled="disabled"></button>`
-//   );
-//   const $helpIcon = newElement(helpInfoIcon);
-//   setDownloadButtonHtml($dlButton, 0, true);
-//   $app.appendChild($dlButton);
-//   $app.appendChild($helpIcon);
-//   $app.appendChild($grid);
-//
-//   /**
-//    * Update the ag-grid container element height on window resize to fill out
-//    * the available vertical space
-//    */
-//   const updateHeight = () => {
-//     const BOTTOM_PADDING = 20;
-//     const top = $grid.children[0].getBoundingClientRect().top;
-//     let height = window.innerHeight - top - BOTTOM_PADDING;
-//     // prevent table from getting too small
-//     if (height < 300) {
-//       height = 300;
-//     }
-//     $grid.style.height = `${height}px`;
-//   };
-//   window.addEventListener("resize", updateHeight);
-//   /**
-//    * Set `grid` to initialized ag-grid Grid for access to Grid API.
-//    * @type {Grid}
-//    */
-//   grid = initAgGrid($grid, HEADERS, singleSampleOutputs, $dlButton);
-//   // initialize height of ag-grid table
-//   updateHeight();
-// }
-//
-// // init table on page load with outputs shared to project
-// getTableData(window.PAGE.page === "shared");
+import { Grid } from "ag-grid-community";
+import { formatDate } from "../../utilities/date-utilities";
+import { escapeHtml, newElement } from "../../utilities/html-utilities";
+import { download } from "../../utilities/file.utilities";
+import {
+  getPrincipalUserSingleSampleAnalysisOutputs,
+  getProjectAutomatedSingleSampleAnalysisOutputs,
+  getProjectSharedSingleSampleAnalysisOutputs,
+  prepareAnalysisOutputsDownload
+} from "../../apis/analysis/analysis";
+import { getIridaWorkflowDescription } from "../../apis/pipelines/pipelines";
+
+import "ag-grid-community/dist/styles/ag-grid.css";
+import "ag-grid-community/dist/styles/ag-theme-balham.css";
+
+const helpInfoIcon = `<i class="fa fa-2x fa-question-circle spaced-left__sm text-info" title="${escapeHtml(
```

Comment:
Instead of this maybe we should create a component for this? Which accepts an optional title and returns the above

Reply:
This is peters code.  I have another branch that I am working on refactoring all of this.

Reply:
Perfect. Thanks